### ISSUE TITLE
imageVersions not creating resized images, but no error either

### ISSUE THREAD

**Issue by huntonas** [tomitrescak/meteor-uploads#72]
I am trying to create different image sizes. I am not getting any errors at all, and the folders are being created for the different sizes. But after uploading the image, there are no images in the created folders. There is the original image in the uploads folder, but the other folders are empty. Any ideas?

    imageVersions: {
			thumbnail: {
				height: 150
			},
			medium: {
				height: 300
			},
			large: {
				height: 450
			}
		}

**Comment by tomitrescak** [comment]
did you install imagemagick on your server?

**Comment by huntonas** [comment]
I do. Just checked and it is there. 

   > identify
Version: ImageMagick 6.7.7-10 2014-03-08 Q16 http://www.imagemagick.org

**Comment by huntonas** [comment]
I am trying to see if this works on a different server, but when I moved it over to Digitalocean using Meteor up, I am not getting and error when trying to upload images. I posted on issue #40. I tried the things listed within that thread, but still getting NaN% in the upload field when I try to upload an image.

**Comment by tomitrescak** [comment]
@huntonas these are really deployment problems that I can't really control. It does not ring a bell. If all is installed well, it should work. Please let me know if you advanced with your problem and post a possible solution for others to see.

**Comment by huntonas** [comment]
@tomitrescak I fixed the error. Seems mup creates a new user for the application and I needed to give that user the permissions for the uploads folder. However, I am still not getting resized images when I upload an image. The folders are being created on the server, but they are empty. I am also not seeing any errors in the logs. I ran sudo apt-get install imagemagick on the server, and it was already there. So I am not sure what is going on. My guess is that I have not configured something correctly in the server/init. From reading the docs, do I need to do anything besides just defining the image sizes that I want in imageVersions : {}?
